src/make-comment.ts

```diff
@@ -33,25 +33,25 @@ const makeComment = async (commentData: CommentData): Promise<void> => {
     });
 
     const botComment = comments.data.find((comment) => comment.body?.includes('<!-- coverage-action-comment -->'));
-
-    const message = await constructComment(commentData);
+    const customMessage = getInput('customMessage');
 
     console.log(botComment);
     console.log(message);
+    console.log(commentData);
 
     if (!botComment) {
       octokit.issues.createComment({
         owner: owner,
         repo: repo,
         issue_number: pullRequestNumber,
-        body: message,
+        body: customMessage === 'comment' ? message : await constructComment(commentData),
```

Comment:
should we let the user of the action know that they can change the action to accept `comment` as customMessage in the workflow ?  even just a sentence in the readme. maybe something like:

This action also can accept a markdown comment .  Modify your workflow like so:

```
customMessage:
     description: toggle if the coverageEndpoint expects to respond with the custom message or just the difference
     default: 'comment'
     required: false
```

for the action to display a comment sent to it by your api.